components/user_settings/notifications/email_notification_setting/email_notification_setting.test.tsx

```diff
@@ -168,6 +168,7 @@ describe('components/user_settings/notifications/EmailNotificationSetting', () =
 
     test('should derived state from props', () => {
         const nextProps = {
+            enableEmail: true,
```

Comment:
Here you are changing the behavior of the test (which may explain why you had to change INTERVAL_NEVER to INTERVAL_INMEDIATE). When we left it as undefined, it was used as false. Later in line 187 was set to true to test a different thing.
TL;DR: here enableEmail should be false.